modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/index.md

```diff
@@ -23,6 +23,10 @@ Surrogate Models:
 - Example 12: [examples/gaussian_process_surrogate.md]
 - Example 13: [modules/stochastic_tools/examples/cross_validation.md]
 
+Bayesian UQ:
+
+- Example 14: [modules/stochastic_tools/examples/bayesian_uq_1d_diff.md]
```

Comment:
As discussed in the stochastic Slack channel, can you please remove the example numbers for all the examples?

Reply:
> As discussed in the stochastic Slack channel, can you please remove the example numbers for all the examples?

Okay!